app/assets/javascripts/webauthn.js

```diff
@@ -22,6 +22,18 @@
     }
   };
 
+  var handleHtmlResponse = function(submit, responseError, response) {
+    if (response.redirected) {
+      window.location.href = response.url;
+    } else {
+      response.text().then(function (html) {
```

Comment:
Nit: There typically shouldn't be any space between the keyword `function` and the parentheses. In other words, `function(html) {` would be the correct syntax.

You likely used the `handleJsonResponse` function as a reference point. It looks like Line 17 and 19 have the funky syntax too 😅 . In `handleHtmlResponse`, Line 29 and 31 needs to be fixed.

On a completely different note, it might be beneficial to at least add a Prettier config to this repo. Setting up a pre-commit hook would be an added bonus but this is a nice-to-have. I can add an issue for this as a bonus side quest.

Reply:
Ah good eye! I added a commit that fixes all those cases in the webauthn.js file https://github.com/rubygems/rubygems.org/pull/3334/commits/6474d63c4e1b321ae8f949849ef5d45d6d5ba0bb Hopefully, the diff isn't too messy.

> it might be beneficial to at least add a Prettier config to this repo. Setting up a pre-commit hook would be an added bonus but this is a nice-to-have. I can add an issue for this as a bonus side quest.

I'm all for adding a linter / formatter for the JS 👍